include/onnxruntime/core/graph/graph_viewer.h

```diff
@@ -38,6 +38,9 @@ class GraphViewer {
   */
   bool GetInitializedTensor(const std::string& tensor_name, const ONNX_NAMESPACE::TensorProto*& value) const;
```

Comment:
can we use update this API to have one more /*out*/ parameter to indicate whether an initializer is overridable please? which should save some API call and look ups.